dhis-2/dhis-support/dhis-support-sql/src/main/java/org/hisp/dhis/db/sql/SqlBuilder.java

```diff
@@ -287,6 +288,16 @@ public interface SqlBuilder {
    */
   String jsonExtractNested(String json, String... expression);
 
+  /**
+   * Generates a database-specific SQL casting expression for the given column or expression.
```

Comment:
We don't have to say "database-specific" in the Javadoc, as everything in `SqlBuilder` is database specific.